jenkins/common/sonic-utilities-build-pr/Jenkinsfile

```diff
@@ -64,22 +64,33 @@ pipeline {
         always {
             junit(allowEmptyResults: true, keepLongStdio: true, testResults: 'sonic-swss-tests/tests/tr.xml')
 
-            publishHTML(target: [
-                allowMissing: false,
-                alwaysLinkToLastBuild: false,
-                keepAll: true,
-                reportDir: 'sonic-utilities/htmlcov',
-                reportFiles: 'index.html',
-                reportName: 'RCov Report'
-            ])
+            script {
```

Comment:
Just a question .. that we need script { } .. ? as we don't run any scripts inside.

Reply:
Yes. `script {}` is required to use conditionals. See: https://github.com/Azure/sonic-build-tools/pull/189